src/app/dashboard/page.tsx

```
@@ -125,63 +118,62 @@ export default function Dashboard() {
             <div className="flex space-x-2">
-              <select className="bg-orange-600 border rounded px-3 py-1 text-white">
+              <select className="bg-orange-600 text-white px-3 py-1 rounded">
                 <option value="Today">Today</option>
                 <option value="This week">This Week</option>
                 <option value="This Month">This Month</option>
-                <option value="Custom Range">Custom Range</option>
               </select>
-              <button className="bg-orange-600 text-white px-3 py-1 rounded text-sm hover:bg-orange-700">
-                Export
-              </button>
+              <button className="bg-orange-600 text-white px-3 py-1 rounded text-sm">Export</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)